<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>592</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>776</width>
    <height>592</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>776</width>
    <height>592</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FormSQL-Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* From uiverse.io by @adamgiebl */
QMainWindow {
	background: rgb(211, 211, 211);
}

/* From uiverse.io by @alexruix */
.QComboBox {
    line-height: 28px;
    border: 2px solid transparent;
    border-bottom-color: #777;
    padding: .2rem 0;
    outline: none;
    background-color: transparent;
    color: #0d0c22;
}

.QComboBox:focus, QComboBox:hover {
    outline: none;
    padding: .2rem 1rem;
    border-radius: 1rem;
    border-color: #7a9cc6;
}

.QLineEdit {
    line-height: 28px;
    border: 2px solid transparent;
    border-bottom-color: #777;
    padding: .2rem 0;
    outline: none;
    background-color: transparent;
    color: #0d0c22;
}

.QLineEdit:focus, QLineEdit:hover {
    outline: none;
    padding: .2rem 1rem;
    border-radius: 1rem;
    border-color: #7a9cc6;
}

.QLineEdit::placeholder {
    color: #777;
}

.QLineEdit:focus::placeholder {
    opacity: 0;
}

QPushButton {
    color: #090909;
    border-radius: 0.1em;
    background: #e8e8e8;
   }
   
   QPushButton:hover {
    border: 1px solid white;
   }
   
 QPushButton:pressed {
    background: rgb(218, 218, 218);
 }</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>12</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type of field:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_TOF">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dropdown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Checkbox</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Date</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txt_Variable">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>34</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>37</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Variable Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Generate">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>10</y>
      <width>138</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_Elements">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>106</y>
      <width>256</width>
      <height>431</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>62</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Add element to the list:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_Code">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>82</y>
      <width>291</width>
      <height>479</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>62</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>62</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Form edit options</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>12</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type of DD:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_TODD">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Yes/No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Element to Checkbox</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Element to Text</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="txt_FEO">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>82</y>
      <width>195</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Copy1">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>52</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Copy2">
    <property name="geometry">
     <rect>
      <x>673</x>
      <y>52</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Element">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>82</y>
      <width>256</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Empty">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Eliminate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Edit">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>540</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cb_TOF</tabstop>
  <tabstop>cb_TODD</tabstop>
  <tabstop>txt_Variable</tabstop>
  <tabstop>txt_Element</tabstop>
  <tabstop>btn_Generate</tabstop>
  <tabstop>txt_FEO</tabstop>
  <tabstop>btn_Copy1</tabstop>
  <tabstop>btn_Copy2</tabstop>
  <tabstop>txt_Code</tabstop>
  <tabstop>lst_Elements</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Copy2</sender>
   <signal>clicked()</signal>
   <receiver>txt_Code</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Copy2</sender>
   <signal>clicked()</signal>
   <receiver>txt_Code</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Copy2</sender>
   <signal>clicked()</signal>
   <receiver>txt_Code</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Copy1</sender>
   <signal>clicked()</signal>
   <receiver>txt_FEO</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Copy1</sender>
   <signal>clicked()</signal>
   <receiver>txt_FEO</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Copy1</sender>
   <signal>clicked()</signal>
   <receiver>txt_FEO</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
